style.css

```diff
@@ -504,3 +871,226 @@
   line-height: 2.75rem;
   color: #fff;
 }
+
+@media screen and (min-width: 768px) {
+  .skills-container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: center;
+    padding-top: 4rem;
+    padding-right: 3rem;
+    padding-left: 2rem;
+  }
+
+  .skills1 {
+    padding-top: 0;
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+
+  .skills2 {
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+
+  .list-items {
+    text-align: start;
+  }
+
+  .list-items li a {
+    font-weight: 100;
+  }
+}
+
+/* form */
+
+.bottom {
+  display: flex;
+  flex-direction: column;
+  padding-top: 7.5rem;
+  padding-bottom: 7.5rem;
+  padding-left: 1.475rem;
+  padding-right: 1.5625rem;
+  gap: 3.62rem;
+}
+
+.contact-bottom-bar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.contact-upper-bar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 6rem;
+  gap: 1.5rem;
+}
+
+.contact-upper-bar p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2.75rem;
+  text-align: center;
+  color: #172b4d;
+}
+
+.input-box {
+  height: 2rem;
+  display: flex;
+  align-items: center;
+  color: #979493;
+  padding: 1.5rem 1.5rem 1.437rem  1.562rem;
+}
+
+.input-box input {
+  border: none;
+  width: 100%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  border-bottom: 1px solid #dbd8d7;
+  font-weight: 400;
+  font-size: 0.94rem;
+  padding: 0.75rem;
+}
+
+.input-area {
+  padding: 1.5rem 1.5rem 1.437rem  1.562rem;
+}
+
+.input-area textarea {
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: italic;
+  font-weight: 400;
+  font-size: 0.9375rem;
+  padding: 0.75rem;
+  background: #fbf8f7;
+  resize: none;
+  border-radius: 0.625rem;
+  height: 7.125rem;
+}
+
+.submit {
+  display: flex;
+  text-decoration: none;
+  border: none;
+  justify-content: center;
+}
+
+.contact-btn {
+  padding: 0.75rem;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.06rem;
+  color: #fff;
+  letter-spacing: 0.03em;
+  border: none;
+}
+
+/* footer */
+.contact_socialmedia {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #081e42 !important;
+  padding-top: 0.935rem;
+  padding-bottom: 2.31rem;
+  cursor: pointer;
+}
+
+.contact_socialmedia,
+.contact_socialmedia-icons {
+  display: flex;
+  flex-direction: row;
+  gap: 1.5rem;
+  color: #081e42 !important;
+}
+
+.footerr {
+  border-top: 1px solid #dfe1e6;
+}
+
+.home-indicator {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-bottom: 0.6875rem;
+}
+
+.indicator {
+  width: 8.375rem;
+  height: 0.3125rem;
+  background: #0d151e;
+  border-radius: 0.625rem;
+}
+
+/* contact responsive design */
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] Please use only one media query for **min-width: 768px** screen size, it seems like you are repeating yourself in the styles at different places which makes it difficult for someone to read through your styles and please make sure that try to use media query less as much as you can. But of course, repeating yourself is not a good idea which actually is considered bad practice 🔴 in the CSS world.